### ISSUE TITLE
UMS.deny feature was broken

### ISSUE THREAD

**Issue by valib** [UniversalMediaServer/UniversalMediaServer#2327]
@SubJunk this is reported in https://www.universalmediaserver.com/forum/viewtopic.php?f=9&t=14510 and was broken in PR 91bc9a6ca179c66aa90ef7a5469e2607fada619e

**Comment by SubJunk** [comment]
Yes we have an issue at https://github.com/UniversalMediaServer/UniversalMediaServer/issues/1801 after we had some comments last year about it being removed https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1623#issuecomment-545119006

If you have time maybe you could look at reverting that change to bring it back?

**Comment by Nadahar** [comment]
As I've explained before somewhere, this was removed from DMS because the implementation was problematic. Since I didn't think it was a frequently used feature, I simply removed it until it could be replaced by a better implementation some time in the future.

The "problem" is rather that you didn't check what you cherry-picked from DMS.

**Comment by SubJunk** [comment]
No one is blaming you, and there is no need to mischaracterise me like that. I did check and I also didn't think anyone used it either, so I was happy to remove it until I realised later that people did.

**Comment by Nadahar** [comment]
@SubJunk  I got the impressions that you weren't aware of this when you merged it the last time this was discussed. There are also many examples of other things that you've merged only to reverse them later, so it's a bit hard for me to believe that you had gone through what you cherry-picked.

That said, if you claim that you knew this and still went ahead, I certainly shouldn't feel the need to explain. In the end it's your responsibility whether you knew about it or not.

**Comment by SubJunk** [comment]
@valib are you interested in doing that reversion? I have a few other things I'm working on in UMS so that would be helpful if you could do it

**Comment by valib** [comment]
@SubJunk I am on holiday so I will check it after New Year's Eve.

**Comment by SubJunk** [comment]
I'm interested in doing this now that we have the web GUI. I'm starting now
